compiler/rustc_trait_selection/src/solve/search_graph/mod.rs

```diff
@@ -109,10 +109,10 @@ impl<'tcx> SearchGraph<'tcx> {
     }
 
     /// We cannot simply store the result of [super::EvalCtxt::compute_goal] as we have to deal with
-    /// coinductive cycles.
+    /// continuously cycles.
     ///
-    /// When we encounter a coinductive cycle, we have to prove the final result of that cycle
-    /// while we are still computing that result. Because of this we continously recompute the
+    /// When we encounter a continuously cycle, we have to prove the final result of that cycle
```

Comment:
The coinductive word has a meaning in this context, so this is not a typo. The replacement you propose does make even less sense.